server/rest/src/main/java/org/infinispan/rest/resources/ContainerResource.java

```diff
@@ -328,7 +328,7 @@ private CacheInfo getCacheInfo(RestRequest request, EmbeddedCacheManager cacheMa
       if (ignoredCaches.contains(cacheName)) {
          cacheInfo.status = "IGNORED";
       } else {
-         if(cacheHealth != HealthStatus.FAILED) {
+         if(cacheHealth != HealthStatus.FAILED && cacheHealth != HealthStatus.DEGRADED) {
```

Comment:
This looks out of place. Is this a different bug?

Reply:
I ended up adding a new status. The problem here is that the cache was not running, which throws an exception.